functorch/_src/aot_autograd.py

```diff
@@ -387,6 +388,12 @@ def fake_fn(primals, tangents):
     with torch.no_grad():
         with track_graph_compiling("joint"):
             fw_module, bw_module = aot_config.partition_fn(fx_g, joint_inputs)
+            fw_outs = [n for n in fw_module.graph.nodes if n.op == "output"][0].args[0]
```

Comment:
@Chillee I hate that there is no built in API for getting the unique(!) output node lol

Reply:
yea we should add that